yt_dlp/extractor/trovo.py

```diff
@@ -43,7 +43,14 @@ def _extract_streamer_info(self, data):
 
 
 class TrovoIE(TrovoBaseIE):
-    _VALID_URL = TrovoBaseIE._VALID_URL_BASE + r'(?!(?:clip|video)/)(?P<id>[^/?&#]+)'
+    _VALID_URL = TrovoBaseIE._VALID_URL_BASE + r'(?:s/)?(?!(?:clip|video)/)(?P<id>(?!s/)[^/?&#]+(?![^?]+\?vid))'
```

Comment:
This doesn't handle `?foo=bar&vid=...`. You can fix the regex, but better to override `suitable` imo